### PR TITLE
Adding 'Advanced Search' to Website and slight reorganization

### PR DESCRIPTION
**Pull request by morgantaschuk**
This is kind of a mess with whitespace. Apologies. A working version is up here: http://morgantaschuk.com/miso-lims/

- Removed docs/_site because this is built automatically by Github Jekyll and added _site to .gitignore
- moved Advanced Search page from wiki to website
-  Renamed 'admin manual' to 'Building and Deploying' and 'value type data' to 'Admin Manual'. 
- Reorganized the nav bar at the left by putting 'order' in the front matter
- Removed some outdated pages from the nav bar by removing the category in the front matter
- added alt text to MISO bowl
- fixed a bunch of broken links

### CODE REVIEW

File: docs/_posts/2016-01-11-installation-guide.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
 layout: page
-title: "Administrator's Manual"
+title: "Installation Guide"
```
Comment (callunity) [inline docs/_posts/2016-01-11-installation-guide.md]: Is there a different term that would make it clear that this also contains instructions for upgrading MISO? (That's the `Releasing and Upgrading` section near the bottom. Maybe moving `Upgrading MISO to a new version` to its own page?)

File: docs/_posts/2018-02-16-advanced-search.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  <td name='samples'></td>
+  <td name='libraries'>🔵</td>
+  <td name='dilutions'>🔵</td>
+  <td name='pool'>🔵</td>
+  <td name='run'>🔵</td>
+  <td name='container'>🔵</td>
+  <td name='ordercomp'>🔵</td>
+</tr>
+<tr>
+  <td name='syntax'>class:<a href="#name">NAME</a></td>
```
Comment (callunity) [inline docs/_posts/2018-02-16-advanced-search.md]: Please highlight that the search terms `class`, `institute`, and `external` are all Detailed Sample only.

File: docs/_posts/2016-01-11-installation-guide.md (hunk 1 above)
Comment (morgantaschuk) [reply docs/_posts/2016-01-11-installation-guide.md]: I can also put it in the new Admin manual.

Comment (morgantaschuk) [reply docs/_posts/2016-01-11-installation-guide.md]: After reading it again, I think you're right and it should go into its own post.

Comment (morgantaschuk) [reply docs/_posts/2016-01-11-installation-guide.md]: Re-reading the docs, would it be better as "Build Guide" / "Deployment guide" / "Admin manual"?

File: docs/_posts/2018-02-16-advanced-search.md (hunk 2 above)
Comment (morgantaschuk) [reply docs/_posts/2018-02-16-advanced-search.md]: I found a new emoji to signal this and put it into the legend.

**Comment by morgantaschuk** [comment]
@callunity, @apmasell, can you review again? http://morgantaschuk.com/miso-lims/ Thanks.

**Comment by callunity** [comment]
Ooh belated request - could we update the types of Illumina platforms on the main page to include the NextSeq and NovaSeq? We could also include the HiSeqs 3000 and 4000 so long as they don't have any funny requirements.Array scanners could go in there too.

**Comment by morgantaschuk** [comment]
Travis build was failing but it wasn't me! So I squashed and merged.

**Pull request merged**